zq2/docs/nodes/nodefaq.md

```diff
@@ -0,0 +1,47 @@
+---
+id: nodes/nodefaq
+title: Frequently Asked Questions
+keywords:
+  - FAQ
+  - Questions
+  - NODE 
+description: Frequently asked questions
+---
+
+# Frequently Asked Questions
+## [Validator node FAQ](#validator-faq)
+
+## How do I sync my validator node? Should I use genesis sync or checkpoint sync ?
+
+Currently, only genesis mode is supported for syncing validator nodes.
+We are actively working on checkpoint sync support feature for validator node. It will be activated through a
+network upgrade coming soon.
+
+## [Non validator node FAQ](#non-validator-node-faq)
+## How do I sync my node if I'm not a validator node?
+For non-validator node, both genesis sync and checkpoint sync option are supported.
+
+## My node is stuck at a specific block and is not progressing. What should i do ?
+Try restarting the docker container, and the node should resume syncing from the last block.
+```bash
+docker restart <container_id>
+```
+## My node starts, but the Docker container stops immediately or after sometime.
+This happens when running the api node from checkpoint.We observed that when a node is started from a
+checkpoint , because of network latency issues, some nodes may experience node crash problem.
```

Comment:
let's remove line 30 and 31 entirely:
```
This happens when running the api node from checkpoint.We observed that when a node is started from a
checkpoint , because of network latency issues, some nodes may experience node crash problem.
```